### PR TITLE
remove resource pages and child models

### PR DESCRIPTION
**Pull request by scytacki**
Resource pages are not really used by project portals. Interactions does have 14 resource pages, but they are used as a way to upload files so the files can be used by activities. The team now uses Cyberduck or some other tool to upload files directly to their S3 bucket.

This PR removes all of the resource page support in the portal. This consists of 3 models:
- ResourcePage
- StudentView
- AttachedFile
The two additional models are only used by resource pages so they can be deleted too.

Additionally I deleted some themes that aren't needed anymore

### CODE REVIEW

**Comment by scytacki** [comment]
@knowuh or @pjanik would you mind skimming this?  
It is mostly just removing things.

**Comment by dougmartin** [comment]
Looks good to me.  That is a lot of code removed!

File: app/models/external_activity.rb [hunk 1]
```diff
@@ -121,12 +121,6 @@ def valid_url
   scope :by_user, proc { |u| { :conditions => {:user_id => u.id} } }
 
   scope :ordered_by, lambda { |order| { :order => order } }
-  # FIXME: See comments in app/models/resource_page.rb
-  scope :match_any, lambda { |scopes|
```
Comment (knowuh) [inline app/models/external_activity.rb]: To check:  Nothing uses ExternalActivity.match_any

File: db/schema.rb [hunk 2]
```diff
@@ -2570,7 +2534,6 @@
 
   add_index "taggings", ["tag_id"], :name => "index_taggings_on_tag_id"
   add_index "taggings", ["taggable_id", "taggable_type", "context"], :name => "index_taggings_on_taggable_id_and_taggable_type_and_context"
-  add_index "taggings", ["tagger_id", "tagger_type"], :name => "index_taggings_on_tagger_id_and_tagger_type"
```
Comment (knowuh) [inline db/schema.rb]: The removal of this index seems unrelated?

**Comment by knowuh** [comment]
This looks good.  Two items I wasn't sure about. 

LOVE all the vanished code.  :+1:

File: app/models/external_activity.rb (hunk 1 above)
Comment (knowuh) [inline app/models/external_activity.rb]: Confirmed it myself:  Nothing uses `#match_any` scope.

**Comment by scytacki** [comment]
Thanks for looking at it!
@knowuh I added back in those indexes, I'm not sure why they were removed.
Yes I removed `match_any` because I couldn't find it used anywhere either.

**Comment by scytacki** [comment]
this was passing all of the tests before and the push is passing the tests. So I'm pretty sure the failures are random. I'm going to merge this and get it over with.
